core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -96,6 +96,18 @@ class CityInfo {
                     cityConstructions.addBuilding(freeBuildingName)
             }
         }
+        
+        // Add buildings and pop we get from starting in this era
+        val ruleset = civInfo.gameInfo.ruleSet
+        val startingEra = civInfo.gameInfo.gameParameters.startingEra
+        if (startingEra in ruleset.eras) {
+            population.setPopulation(ruleset.eras[startingEra]!!.settlerPopulation)
+            for (building in ruleset.eras[startingEra]!!.settlerBuildings) {
+                if (ruleset.buildings[building]!!.isBuildable(cityConstructions)) {
+                    cityConstructions.addBuilding(building)
+                }
+            }
+        }
```

Comment:
This looks like it applies to ALL cities that we'll found in the game. Is that how it works in Civ V? I really don't know, I don't think I ever started on a later era

Reply:
yup it applies to all cities

Reply:
I haven't tested this in the original, but I remember reading somewhere (probably the fandom or some random forum post?) that it applies to all cities, even those not founded by the originally spawned settlers